grails-app/domain/org/pih/warehouse/order/OrderItem.groovy

```diff
@@ -138,8 +138,8 @@ class OrderItem implements Serializable, Comparable<OrderItem> {
         requestedBy(nullable: true)
         quantity(nullable: false, min: 1)
         quantityUom(nullable: true)
-        // Validate quantity per uom != 0 (value as boolean, to catch 0, 0.0 and '')
-        quantityPerUom(nullable: false, validator: { value -> value as boolean } )
+        // Validate quantity per uom > 0 (value as boolean, to catch 0, 0.0 and '')
+        quantityPerUom(nullable: false, validator: { value -> value as boolean && value > 0 } )
```

Comment:
It probably doesn't hurt to keep it but do we still need the `value as boolean` part? I assume nullable:false handles empty string and null value but even if it doesn't I have a sneaking suspicion that it would be handled implicitly by the `value > 0` custom validation logic. I will check this using unit tests but I think we can get rid of this part of the logic.

Reply:
@jmiranda Yup, you're right, the `as boolean` was for `non 0/empty/null`, since we now have `>= 1`, then it's not needed anymore.